src/bluehawk/parser/RootParser.ts

```diff
@@ -53,8 +53,11 @@ annotatedText
 attributeList
   : AttributeListStart (attributeList | JsonStringLiteral | Newline | LineComment)* AttributeListEnd
 
+blockCommandUncommentedContents
+  : BlockCommentEnd Newline (chunk)* BlockCommentStart 
+
 blockCommand
-  : (LineComment)? CommandStart (commandAttribute)? Newline (chunk)* (LineComment)* CommandEnd
+  : (LineComment)? CommandStart (commandAttribute)? (BlockCommentEnd*backtrack* | (Newline)?) ((chunk)* | blockCommandUncommentedContents) CommandEnd
```

Comment:
I just noticed this seems to be out of date with the actual parser below:
```suggestion
  : CommandStart (commandAttribute)? (BlockCommentEnd*backtrack* | (Newline)?) ((chunk)* | blockCommandUncommentedContents) CommandEnd
```